tests/keras_tests/function_tests/test_activation_quantization_functions.py

```diff
@@ -0,0 +1,64 @@
+# Copyright 2024 Sony Semiconductor Israel, Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+
+import unittest
+
+import numpy as np
+
+from model_compression_toolkit.constants import THRESHOLD, SIGNED, RANGE_MIN, RANGE_MAX
+
+from model_compression_toolkit.core.keras.quantizer.fake_quant_builder import symmetric_quantization, \
+    uniform_quantization
+
+
+class TestActivationQuantizationFunctions(unittest.TestCase):
```

Comment:
How did we miss writing tests to this? 
Isn't it tested in test_quantization_configurations.py? (if not, maybe we should add it there instead of in this separate test)

Reply:
You're correct. test_quantization_configurations checks it, but it's only for Keras (and I added tests for Pytorch). So I can either create a similar test_quantization_configurations for pytorch, or these kind of tests for Keras. I think the first, is a better idea actually.

Reply:
They are not tested in test_quantization_configurations. These functions are used in some cases like MP for activations and SNC (which are all tested only with PoT quantizer, currently).